### ISSUE TITLE
Fix Data Table - 508 issue (TEST ONLY) 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#1494]
## Required Fix 
- [ ] RF-3b
- [ ] RF-5b
- [ ] RF-8

## Context

Development work will be completed in Align Data Tables to USWDS #1361. This ticket is for 508 testing only 

See [EPA CAMD Accessibility Report ](https://teams.microsoft.com/l/file/31E8B6EF-1313-4495-B867-46347BB7E4A2?tenantId=88b378b3-6748-4867-acf9-76aacbeca6a7&fileType=docx&objectUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam%2FShared%20Documents%2FDatateers%2F5-UI%20Design%2F02%20508%20and%20Accessibility%2FEPA.CAMPD.Accessibility.Report%20.docx&baseUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam&serviceName=teams&threadId=19:f4349e2ba95e43f3b29e8208bc8f5d5f@thread.skype&groupId=21a5e1cd-d3f1-48a6-801f-0d34e2d63d23) for how to fix and additional details 

| Issue #  | Issue Description  |
|-|-|
| RF-3b | ARIA is only needed when native html tags are not used. The data table in the Data Preview has both native html tags and aria roles, the aria roles can be removed: |
| RF-5b | The sortable table can only be accessed by a mouse, a keyboard user is unable to tab to and activate the controls. |
| RF-8 | When tabbing through the page, the focus does not follow a logical order which may confuse users. This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 2.4.3 |

**Comment by aprematta** [comment]
[EPA Monitoring Plans Accessibility Report Additional.docx](https://github.com/US-EPA-CAMD/easey-ui/files/6632145/EPA.Monitoring.Plans.Accessibility.Report.Additional.docx)

**Comment by aprematta** [comment]
Adding audit report for most recent 508 review, the table now has 3 outstanding issues, 
RF 3: The sortable table does not have a programatic indicator to tell a screen reader when a column is sorted in ascending or descending order. 
RF-7: The data tables in the application lack a programatic name:<div class="sc-bdnxRM ecDzaP rdt_Table" role="table">
RF-8: There are multiple tags with the same id when data tables are used on a page. For example:
6RmV1OFmK3-undefined
div id="cell-6RmV1OFmK3-undefined" role="gridcell" data-tag="allowRowEvents" class="sc-hKFxyN sc-eCApnc sc-jSFjdj iCxiXr domoiu dUbULe rdt_TableCell"><div data-tag="allowRowEvents">5</div></div>

div id="cell-6RmV1OFmK3-undefined" role="gridcell" data-tag="allowRowEvents" class="sc-hKFxyN sc-eCApnc sc-jSFjdj iCxiXr domoiu dUbULe rdt_TableCell"><div data-tag="allowRowEvents">7</div></div>
